src/parser/monk/mistweaver/modules/spells/Celestial.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'parser/core/Analyzer';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class Celestial extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  petID = null;
+  soothHealing = 0;
+  soothOverHealing = 0;
+  envelopHealing = 0;
+  evelopOverhealing = 0;
+  healing = 0;
+  overhealing = 0;
+  count = 0;
+
+  on_byPlayer_summon(event) {
+    this.petID = event.targetID;
```

Comment:
There can technically be more than one "active" summon for a monk, right? This makes the module only track the most recent summon event, which could technically make a statue or random summon (trinkets, essences, etc.) break the module